.travis.yml

```
@@ -2,8 +2,18 @@ language: python
 dist: xenial
 matrix:
```

`jobs` is replacing `matrix` at Travis CI
```suggestion
jobs:
```